src/main/resources/db.sql

```diff
@@ -1,5 +1,6 @@
+DROP TABLE IF EXISTS quotes;
```

Comment:
Why do you need this? I have used such a script in the past, and since the H2 db is in-memory, it should be created from scratch when the server starts

Reply:
I faced some problems when running tests without such addition. I might guess, that during test process H2 created only once, so not created each time for each test.

Reply:
Can you tell me how to reproduce?

Reply:
You need to have at least two tests which use QuoteOfTheDayVerticle.

Reply:
When deploying the verticle for the first time, it will create an instance of h2, which will be reused for the second test, so you will get something like this:
```
Table "quotes" already exists; SQL statement:
CREATE TABLE quotes (
  quote_id BIGINT AUTO_INCREMENT PRIMARY KEY NOT NULL,
  text VARCHAR NOT NULL,
  author VARCHAR NOT NULL
);
 [42101-196]

	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:117)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.execute(NewProxyStatement.java:75)
	at io.vertx.ext.jdbc.impl.actions.JDBCExecute.execute(JDBCExecute.java:46)
	at io.vertx.ext.jdbc.impl.actions.JDBCExecute.execute(JDBCExecute.java:31)
	at io.vertx.ext.jdbc.impl.actions.AbstractJDBCAction.handle(AbstractJDBCAction.java:58)
	at io.vertx.ext.jdbc.impl.actions.AbstractJDBCAction.lambda$execute$0(AbstractJDBCAction.java:66)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$1(ContextImpl.java:273)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

Reply:
The fastest way to reproduce is to either create a second test in the skeleton repository or simply:
1. clone my fork
2. remove `DROP TABLE IF EXISTS quotes;` line in the `db.sql` file
3. try to run tests

Reply:
@Sammers21 ha I see. Ok then that's fine.